Makefile

```diff
@@ -100,23 +100,24 @@ GO_TEST_FORMAT = pkgname
 DEFAULT_RELEASE_IMAGES = $(shell (tr -d '\n\t ' < ${ROOT_DIR}/data/default_release_images.json))
 DEFAULT_OS_IMAGES = $(shell (tr -d '\n\t ' < ${ROOT_DIR}/data/default_os_images.json))
 
+# Support all Release/OS images for CI
 ifeq ($(CI), true)
 	VERBOSE = true
-	OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell hack/get_ocp_versions_for_testing.sh))
-else
-	OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell hack/get_ocp_versions_for_testing.sh | jq -c 'with_entries(select(.value.default))'))
-	ifneq ($(OPENSHIFT_VERSIONS), {})
-		DEFAULT_VERSION := $(shell (jq -n '$(OPENSHIFT_VERSIONS)' | jq 'keys[0]'))
-		RELEASE_IMAGES := $(or ${RELEASE_IMAGES}, $(shell (echo '$(DEFAULT_RELEASE_IMAGES)' | jq -c --arg v $(DEFAULT_VERSION) 'map(select(.openshift_version==$$v))')))
-		OS_IMAGES := $(or ${OS_IMAGES}, $(shell (echo '$(DEFAULT_OS_IMAGES)' | jq -c --arg v $(DEFAULT_VERSION) 'map(select(.openshift_version==$$v))')))
-	endif
+	RELEASE_IMAGES := $(or ${RELEASE_IMAGES},${DEFAULT_RELEASE_IMAGES})
+	OS_IMAGES := $(or ${OS_IMAGES},${DEFAULT_OS_IMAGES})
 endif
 
-RELEASE_IMAGES := $(or ${RELEASE_IMAGES},${DEFAULT_RELEASE_IMAGES})
-OS_IMAGES := $(or ${OS_IMAGES},${DEFAULT_OS_IMAGES})
+# Support default Release/OS image if lists not specified
+ifndef RELEASE_IMAGES
+ifndef OS_IMAGES
```

Comment:
what will happen if user overrides ``OS_IMAGES`` and not ``RELEASE_IMAGES``?

Reply:
That's the kube-api flow, i.e. specifying only OS_IMAGES. So we handle it in line 120.